playwright.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  use: {
+    video: 'on',
```

Comment:
Technically video is supposed to be off in a sauce vm (https://github.com/saucelabs/sauce-playwright-runner/blob/e8ed632f976fd4e2c3a34be18479ca660805770b/src/utils.js#L79-L81). But there's a bug in the current latest version of the runner that does not respect this (if I remember correctly its because the video param is supposed to be `--param video` but the runner was formatting the arg like `--param video=true/false`). I'm guessing it hasn't blown up in sauce because playwright includes its own ffmpeg? Not totally sure.

Anyways, just wanted to point this out. Not sure it necessarily needs to be addressed here since templating this config might not be worth the trouble right now.

Reply:
@mhan83 I disabled the video recording when running in Sauce VMs !